test/controllers/SwitchLanguageControllerSpec.scala

```diff
@@ -74,4 +75,18 @@ class SwitchLanguageControllerSpec extends WordSpec with Matchers with GuiceOneA
     }
   }
 
+  "Lang swtich" should {
+    "redirect to current page if referer is valid" in {
+      val result = controller.switchToLanguage("cy")(fakeRequest.withHeaders((HeaderNames.REFERER, s"${appConfig.baseHostUrl}/somepage")))
+      redirectLocation(result) shouldBe Some(s"${appConfig.baseHostUrl}/somepage")
+      confirmLangCookie(cookies(result), "en")
```

Comment:
Are we moving to Welsh here? Should the cookie value be "cy"?

Reply:
As its an override and likely to be fixed in the library, better to leave as much as found in order to see the changes. I think Intelij is wrong and the line itself would be better without the IF ;)

Reply:
Lines removed